be/src/olap/tablet.cpp

```diff
@@ -3396,8 +3408,9 @@ void Tablet::calc_compaction_output_rowset_delete_bitmap(
         for (uint32_t seg_id = 0; seg_id < rowset->num_segments(); ++seg_id) {
             src.segment_id = seg_id;
             DeleteBitmap subset_map(tablet_id());
-            input_delete_bitmap.subset({rowset->rowset_id(), seg_id, start_version},
-                                       {rowset->rowset_id(), seg_id, end_version}, &subset_map);
+            input_delete_bitmap.subset_ignore({rowset->rowset_id(), seg_id, start_version},
```

Comment:
It can't resolve the compaction issue, since the compaction use merge-on-read process to dedup keys, it can't identify which keys should be ignored.